### DISCUSSION TITLE
PDF Attachments in WhatsApp

### DISCUSSION THREAD

**Discussion by skeelkat** [ferdium/ferdium-app#1432] [Q&A]
Is it possible by any means to open PDF documents in the WhatsApp Services be set to open directly within Ferdium instead of trying to download and save the file? I receive quite a lot of PDF's daily but do not need to save them all to disk and then open with external editor.

WhatsApp Web and the WhatsApp App have this function.

I tried enabling the "Open URL's within Ferdium" but this has no effect.

**Comment by NR2014** [comment]
+1
